detection.py

```diff
@@ -100,3 +102,44 @@ def detect_drift(
         return self._compare_two_numerical_distributions(
             baseline, new, test=numerical_test
         )
+
+    def _remove_missing_values(
+        self,
+        baseline: Union[list, np.ndarray, pd.Series],
+        new: Union[list, np.ndarray, pd.Series],
+        base_cat_freq: OrderedDict,
+        new_cat_freq: OrderedDict
+    ) -> Tuple[OrderedDict, OrderedDict]:
+        """
+        This method gets the missing values frequencies from the original arrays and compares missing values frequencies
+        :param baseline: Baseline (old period / train set) vector
+        :param new: Target (future period / test set) vector
+        :param base_cat_freq: Baseline vector value counts with missing values
+        :param new_cat_freq: Target vector value counts with missing values
+        :type base_cat_freq: object without missing values
+        :type new_cat_freq: object without missing values
```

Comment:
:type is to indicate the data type, you can remove (they appear already as typing hints)